Source/Blazorise/ComponentDisposer.cs

```diff
@@ -62,7 +62,8 @@ public ComponentDisposer( IServiceProvider serviceProvider )
 
         public void Dispose<TComponent>( TComponent component ) where TComponent : IComponent
         {
-            if ( !active ) return;
+            if ( !active )
+                return;
```

Comment:
hehe... see line 74... I had changed the other condition to be single line also...

Reply:
I'm blind 😅